packages/storage/amplify_storage_s3_dart/test/utils/app_path_provider_test.dart

```diff
@@ -0,0 +1,38 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:io';
+
+import 'package:amplify_storage_s3_dart/src/utils/app_path_provider.dart';
+import 'package:test/test.dart';
+
+// ignore: invalid_annotation_target
+@TestOn('vm')
```

Comment:
Is this working? I thought this had to be the very first thing listed in the file (besides comments) for it to be recognized, but maybe not? I would put it above the imports anyway just for consistency. It is where we place it in other tests.

Reply:
Nope this not working correctly, have fixed locally but forgot to push the commit here. The tag should be place on top of the file 😅